hooks/boot/13-tuning.rb

```diff
@@ -1,7 +1,7 @@
 if has_custom_fact?('tuning')
   # TODO: automatically get the tuning sizes - standard doesn't exist and only
```

Comment:
Looks like this is out of date
```suggestion
  # TODO: automatically get the tuning sizes - default doesn't exist and only
```